back_end/database/update_db_helpers.py

```diff
@@ -161,7 +182,7 @@ def parse_registrar_course_data(course: json) -> dict:
             "audit": False,
         },
         "NPD": {"pdf": {"required": False, "permitted": False}, "audit": True},  # npdf
```

Comment:
Sorry don't mean to be nit-picky because it doesn't really matter much, but for style consistency, shouldn't this also be on separate lines like the other mappings?

Reply:
yeah, i applied a python formatter to my `.py` files and the formatter automatically styled the `NPD` mapping differently!